src/core.e/Complex.pm6

```diff
@@ -0,0 +1,5 @@
+augment class Complex {
+    method sign() { self / self.abs }
```

Comment:
You need a check for 0 here. sign(0+0i) should be 0, not trigger a division-by-zero error.